### ISSUE TITLE
Add possibility to search for new APK-updates like from APKMirror.com

### ISSUE THREAD

**Issue by AnonymousWP** [SmartPack/PackageManager#98]
A current repository already exists, but its development is stalled: https://github.com/hemker/apkgrabber. It'd be nice to clone the repo and/or implement the necessary code.

**Comment by sunilpaulmathew** [comment]
@AnonymousWP 
I am afraid, it will lead to removal of my app from Google Play as it is against the Google's policies.

**Comment by AnonymousWP** [comment]
> @AnonymousWP I am afraid, it will lead to removal of my app from Google Play as it is against the Google's policies.

Fair point, but what you can do is maintain two paths: the one with the built-in downloader (you can just open an external browser to APKMirror.com) you publish it on GitHub. The one without via Google Play. I think that should be legally fine.

**Comment by sunilpaulmathew** [comment]
> > @AnonymousWP I am afraid, it will lead to removal of my app from Google Play as it is against the Google's policies.
> 
> Fair point, but what you can do is maintain two codebases: the one with the built-in downloader (you can just open an external browser to APKMirror.com) you publish it on GitHub. The one without via Google Play. I think that should be legally fine.

I really don't want to spend time for maintaining an extra code base. Sorry
